lib/score_tracker.rb

```diff
@@ -18,6 +18,7 @@ def self.is_a_row_full(board, character)
 
   def self.extract_rows(board)
```

Comment:
This implementation is tied to a 3x3 board, but if the board size changes then your implementation changes. The same goes as how you are letting users interface with a board. Your labelling a cell on the board to a specific character. If the board size changes that can get hard to manage.